docs/remote-dev.md

```diff
@@ -0,0 +1,68 @@
+# Visual Studio Code Remote Development
+
+[VSCode Remote Development](https://code.visualstudio.com/docs/remote/remote-overview) allows you to run your code and environment in a container. This is especially useful for Swift when developing on macOS and deploying to Linux to ensure there are no compatibility issues in Foundation when running your code.
```

Comment:
This is probably a better link https://code.visualstudio.com/docs/remote/containers as it talks specifically about containers.

Reply:
"This is especially ... running your code" is a very long sentence, maybe it can be broken up.